File: templates/python/class/___name___.py
from typing import *
from . import * 
class {{ name }}({% if inherit != ""%}{{ inherit }}{%else%}object{%endif%}):
    def __init__(self{%if create_parametrized_constructor%}{% for field, type in fields.items()%}, {{field}}: {{ type }}{%endfor%}{%endif%}):
        {% for field, type  in fields.items() %}
        self.{{ field }} = {% if create_parametrized_constructor%}{{ field }}{%else%}None{%endif%}
        {% endfor %}
        pass
    {% for method, type in methods.items() %}
    def {{ method }}(self, {% for param, ptype in type.items() %}{% if param != "return"%}{{ param}}: {{ ptype}}{% if not loop.last%}, {%endif%}{%endif%}{% endfor %}) -> {{ type["return"] }}:
        pass
    {%endfor %}  

File: templates/java/class/___name___.java
package {{ package }};

import java.util.Date;

public class {{ name }} {
	
	{% for field in fields %}
	private {{ field["type"] }} {{ field["name"] }}; 
	{% endfor %}
	public {{ name }}() 
	{
	}
	public {{ name }}({% for field in fields %}{{ field["type"] }} _{{ field["name"] }}{% if not loop.last %}, {% endif %}{% endfor %}) {
		super();
		{% for field in fields %} 
		this.{{ field["name"] }} = _{{ field["name"] }};
		{% endfor %}
	}
	
	{% for field in fields %} 
	public {{field["type"]}} get{{field["name"][0].capitalize() + field["name"][1:] }}() 
	{
		return this.{{ field["name"] }};
	}

	public void set{{field["name"][0].capitalize() + field["name"][1:] }}({{ field["type"] }} newValue) 
	{
		this.{{ field["name"] }} = newValue;
	}
	{% endfor %}
}
